### PR TITLE
Add FAQ: cert_recycle_bin

### PR DESCRIPTION
**Pull request by larswmh**
DE:
image

EN:
image

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -224,6 +224,17 @@
...
+                            "<b>Android:</b><ul><li>Tippen Sie unter Status (vor Version 2.12 'Startseite') auf das Drei-Punkte-Menü oben rechts in der Ecke.</li><li>Tippen Sie daraufhin auf 'Papierkorb'.</li><li>Tippen Sie innerhalb eines der dort aufgelisteten Zertifikate auf das Drei-Punkte-Menü oben rechts.</li><li>Wählen Sie aus, ob Sie das ausgewählte Zertifikat wiederherstellen oder endgültig aus dem Papierkorb entfernen wollen.</li></ul>",
+                            "Zur Veranschaulichung können Sie unter den folgenden Links Screenshots für das Warnen in Vertretung für sowohl iOS als auch Android Geräte finden:<ul><li><a href='../screenshots/#ios_recycle_bin' target='_blank'>iOS Screenshots: Papierkorb</a></li><li><a href='../screenshots/#android_recycle_bin' target='_blank'>Android Screenshots: Papierkorb</a></li></ul>"
```
Comment (Ein-Tim) [inline src/data/faq_de.json:236]: Für das Warnen in Vertretung?

```suggestion
                            "Zur Veranschaulichung können Sie unter den folgenden Links Screenshots zum Papierkorb für sowohl iOS als auch Android Geräte finden:<ul><li><a href='../screenshots/#ios_recycle_bin' target='_blank'>iOS Screenshots: Papierkorb</a></li><li><a href='../screenshots/#android_recycle_bin' target='_blank'>Android Screenshots: Papierkorb</a></li></ul>"
```

File: src/data/faq.json [hunk 2]
```diff
@@ -224,6 +224,17 @@
...
+                            "<b>Android:</b><ul><li>While on the Status tab ('Start Screen' before version 2.12), tap the three-dot menu in the top-right corner.</li><li>Tap 'Recycle Bin'.</li><li>Within one of the certificates listed there, tap the three-dot menu in the top-right corner.</li><li>Select whether you want to restore the selected certificate or permanently remove it from the recycle bin.</li></ul>",
+                            "You can find screenshots for warning for others for both iOS and Android devices at the following links:<ul><li><a href='../screenshots/#ios_recycle_bin' target='_blank'>iOS Screenshots: Recycle Bin</a></li><li><a href='../screenshots/#android_recycle_bin' target='_blank'>Android Screenshots: Recycle Bin</a></li></ul>"
```
Comment (Ein-Tim) [inline src/data/faq.json:236]: Warning for others?

```suggestion
                            "You can find screenshots on the recycle bin for both iOS and Android devices at the following links:<ul><li><a href='../screenshots/#ios_recycle_bin' target='_blank'>iOS Screenshots: Recycle Bin</a></li><li><a href='../screenshots/#android_recycle_bin' target='_blank'>Android Screenshots: Recycle Bin</a></li></ul>"

```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Two small change requests.

File: src/data/faq.json (hunk 2 above)
Comment (larswmh) [reply src/data/faq.json:236]: Forgot to change that while copying from another part of the FAQ, thanks!

File: src/data/faq_de.json (hunk 1 above)
Comment (larswmh) [reply src/data/faq_de.json:236]: Forgot to change that while copying from another part of the FAQ, thanks!

**Review by Ein-Tim — APPROVED** [review]
LGTM now! I did not check for typos/etc.

**Comment by dsarkar** [comment]
@Ein-Tim Thanks for the review!

**Pull request merged**